lib/blocks.php

```diff
@@ -180,7 +206,7 @@ function gutenberg_register_core_block_styles( $block_name ) {
 			"wp-block-{$block_name}",
 			gutenberg_url( $style_path ),
 			array(),
-			filemtime( gutenberg_dir_path() . $style_path )
+			$default_version
```

Comment:
@aristath – I copied the logic from other CSS files used in Gutenberg that ensures that the plugin version is used on production.

Reply:
Makes sense :+1: